misc/python/genapi.py

```diff
@@ -40,6 +40,11 @@
     #'mesh send',
 ]
 
+# API Version:
+VERSION = '2.0a1'
+# minimega version (set this before rendering)
+MM_VERSION = 'UNKNOWN'
```

Comment:
See https://github.com/sandia-minimega/minimega/blob/master/build.bash#L8

We auto-generate the version as the commit hash, and hand-tool a version number on releases. Could we do that here as well?